docs/dev/marketplace_offering.md

```diff
@@ -0,0 +1,118 @@
+
+  - Training and Support for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.
+
+  - Exclusive opportunities to sponsor conferences and events (such as Datadog DASH) at a discounted rate.
+  
+  - Generating new leads from user discovery
+
+{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}
+
+## Join the Datadog partner network
+
+Before listing an offering on the Datadog Marketplace, you will first need to apply to the [Datadog Partner Network's][3] **Technology Partner** track. Once your application has been approved, you can begin to develop your offering.
+
+## Request a sandbox account
+
+All Technology Partners can request a dedicated sandbox Datadog account to aid in their development.
+
+To request a sandbox account:
+
+1. Log into the [Datadog Partner Portal][6].
+2. On your personal homepage, click on the **Learn More** button under **Sandbox Access**.
+3. Select **Request Sandbox Upgrade**.
+
+<div class="alert alert-info">If you are already a member of a Datadog organization (including a trial org), you may need to switch to your newly created sandbox. For more information, see the <a href="https://docs.datadoghq.com/account_management/org_switching/">Account Management documentation</a>.</div>
+
+Creating a developer sandbox may take up to one or two business days. Once your sandbox is created, you can [invite new members from your organization][7] to collaborate with.
+
+## Explore learning resources
+
+Once you've joined the Technology Partner track and requested a sandbox account, you can start learning about developing offerings by:
+
+* Completing the on-demand [**Introduction to Datadog Integrations**][8] course on the [Datadog Learning Center][9].
+* Reading the documentation about setting up an [OAuth 2.0 client][11] for API-based integrations.
+
+
+### Request access to Marketplace
+
+To request access to the private Marketplace repository, email <a href="mailto:marketplace@datadoghq.com">marketplace@datadoghq.com</a>. Once you have been granted access, you can review an [example pull request][12] in the Marketplace repository with annotations and best practices.
+
+## Coordinate GTM opportunities
+
+Once a Marketplace tile is live, Technology Partners can meet with Datadog's Partner Marketing team to coordinate a joint go-to-market (GTM) strategy, which includes the following:
+
+- A Datadog quote for partner press releases
+- A blog post on the [Datadog Monitor][23]
+- Amplification of social media posts
+
+## Getting Started
+To get started with creating an offering, see [Create a Tile][13]. If you're interesting in building an Agent-based integration and selling it on the Datadog Marketplace, see [Create an Agent-based Integration][19].
```

Comment:
Should we also list out building API-based integration?

Reply:
Clarified in upcoming commit